prac_06/guitar_test.py

```diff
@@ -0,0 +1,19 @@
+# Adrianto Hartono
+# 14347930
+
+"""
+Guitar Test
+
+Expect: 15 min
+Actually: 9 min
+"""
+
+from prac_06.guitar import Guitar
+
+gibson_guitar = Guitar("Gibson L-5 CES", 1922, 16035.40)
```

Comment:
I think before this yon should add def function

Reply:
okay, thank you for the feedback 🙏